halo2-base/src/gates/range.rs

```diff
@@ -164,6 +209,9 @@ pub trait RangeInstructions<F: ScalarField> {
     );
 
     /// Checks that `a` is in `[0, b)`.
```

Comment:
I guess it's clear from the function but can add that `b` is `u64`. BTW is `b = u64::MAX` safe?